boxoffice/views/admin.py

```diff
@@ -40,22 +41,35 @@ def org(organization):
     return dict(org=organization, title=organization.title)
 
 
+@app.route('/api/1/admin/o/<org>/items')
+@lastuser.requires_login
+@load_models(
+    (Organization, {'name': 'org'}, 'organization'),
+    permission='org_admin'
+    )
+@xhr_only
+@requestargs('search')
+def admin_items(organization, search=None):
+    if search:
+        filtered_items = Item.query.join(ItemCollection).filter(Item.title.ilike('%{query}%'.format(query=search))).all()
+        return jsonify(items=[{'id': str(item.id), 'title': item.title} for item in filtered_items])
```

Comment:
Can we use `api_success` and `api_error` functions here as well? `api_error` if the search parameter is `None`.